components/Admissions/EventItem.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import moment from "moment";
+import { Box, Flex, Text, Image, Link } from "native-base";
+
+function EventItem(
+  id: number,
+  title: string,
+  cover: string,
+  location: string,
+  eventType: string,
+  startTime: string
+) {
+  interface EventType {
+    [key: string]: string;
+  }
+
+  const EVENT_CONSTANTS: EventType = {
+    company_presentation: "Bedriftspresentasjon",
+    lunch_presentation: "Lunsjpresentasjon",
+    alternative_presentation: "Alternativ bedpres",
+    course: "Kurs",
+    breakfast_talk: "Frokostforedrag",
+    party: "Fest",
+    social: "Sosialt",
+    event: "Arrangement",
+    kid_event: "KID-arrangement",
+    other: "Annet",
+  };
+
+  const eventTypeToString = (eventT: string) => {
+    return EVENT_CONSTANTS[eventT] || EVENT_CONSTANTS.other;
+  };
+
+  return (
+    <Link href={`https://abakus.no/events/${id}`} isExternal key={id}>
+      <Box
+        rounded="8"
+        overflow="hidden"
+        borderWidth="3"
+        borderColor="red.900"
+        width="365"
+        height="260"
+        shadow="30"
+        bg="white"
+        key={id}
+        alignItems="center"
+      >
+        <Image
+          source={{ uri: cover }}
+          alt="Alternate Text"
+          width="100%"
+          height={110}
+        />
+        <Text
+          color="coolGray.800"
+          mt="3"
+          fontWeight="medium"
+          fontSize="xl"
+          textAlign="center"
+        >
+          {title}
+        </Text>
+        <Text mt="2" fontSize="sm" textAlign="center" color="coolGray.700">
+          {`${moment(startTime)
+            .utc()
+            .format("MMM.Do.HH:mm")} ⊙ ${location} ⊙ ${eventTypeToString(
+            eventType
+          )}`}
```

Comment:
Why is this UTC? Should it not be the current time zone?

Reply:
I fixed.